src/types.h

```diff
@@ -997,6 +989,48 @@ struct Stobj
     Vec g_local_vel;
 };
 
+struct Stobj_
```

Comment:
I have to ask, underscore?

Reply:
I didn't realize there was already a Stobj struct when I renamed this one to Stobj, so I did that temporarily because I didn't feel like unifying them all at once. It will get removed soon.